tasks/shared_libraries/recovery/shared_recovery.py

```diff
@@ -5,86 +5,106 @@
     """
     Provides potential actions for the database lambda to take when posting to the SQS queue.
     """
-    NEW = 'post'
-    UPDATE = 'put'
+
+    NEW = "post"
+    UPDATE = "put"
+
 
 class OrcaStatus(Enum):
     """
     Defines the status value used in the ORCA Recovery database for use by the recovery functions.
     """
+
     PENDING = 1
-    STAGED  = 2
+    STAGED = 2
     SUCCESS = 3
-    FAILED  = 4
-
-def post_status_for_job_to_queue(job_id: str, granule_id: str, status_id: OrcaStatus,
-                                 request_time: Optional[str], completion_time: Optional[str],
-                                 archive_destination: Optional[str],
-                                 request_method: RequestMethod, db_queue_url: str):
-    
-    new_data = {'job_id': job_id, 'granule_id': granule_id}
+    FAILED = 4
+
+
+def post_status_for_job_to_queue(
+    job_id: str,
+    granule_id: str,
+    status_id: OrcaStatus,
+    request_time: Optional[str],
+    completion_time: Optional[str],
+    archive_destination: Optional[str],
+    request_method: RequestMethod,
+    db_queue_url: str,
+):
+
+    new_data = {"job_id": job_id, "granule_id": granule_id}
     if request_time is not None:
-        new_data['request_time'] = request_time
+        new_data["request_time"] = request_time
     if completion_time is not None:
-        new_data['completion_time'] = completion_time
+        new_data["completion_time"] = completion_time
     if archive_destination is not None:
-        new_data['archive_destination'] = archive_destination
-
-    post_entry_to_queue('orca_recoveryjob',
-                        new_data,
-                        request_method, db_queue_url)
-
-def post_status_for_file_to_queue(job_id: str, granule_id: str, filename: str, key_path: Optional[str],
-                                  restore_destination: Optional[str],
-                                  status_id: OrcaStatus, error_message: Optional[str],
-                                  request_time: Optional[str], last_update: str,
-                                  completion_time: Optional[str],
-                                  request_method: RequestMethod,
-                                  db_queue_url: str):
-    new_data = {'job_id': job_id,
-                'granule_id': granule_id,
-                'filename': filename}
+        new_data["archive_destination"] = archive_destination
+
+    post_entry_to_queue("orca_recoveryjob", new_data, request_method, db_queue_url)
+
+
+def post_status_for_file_to_queue(
+    job_id: str,
+    granule_id: str,
+    filename: str,
+    key_path: Optional[str],
+    restore_destination: Optional[str],
+    status_id: OrcaStatus,
+    error_message: Optional[str],
+    request_time: Optional[str],
+    last_update: str,
+    completion_time: Optional[str],
+    request_method: RequestMethod,
+    db_queue_url: str,
+):
+    new_data = {"job_id": job_id, "granule_id": granule_id, "filename": filename}
```

Comment:
Add doc string. See previous comments on other items to cleanup in the shared library.

Reply:
@AndrewDorn to provide the doc string